loomi/flutter-leap
# update/flutter-and-plugin-versions

**What I did:**
- Update Flutter version (FVM) to 3.7.10
- Update dart sdk version used to 2.19.6
- Update all plugin's version that had updates available.

**How to test:**
1. **Run the following commands:**
+ flutter clean
+ flutter pub get
+ flutter pub run build_runner build --delete-conflicting-outputs

2. **How to get there**
+ Run the app

**What to test:**
+ Check if the APP runs
+ Check if the stores state are still working
+ Test the things that you think are worth testing, even the ones that are not in this description

**Expected results:**
+ App should run with success
+ Primarily mobx stores must be still working as expected
